src/firefly/java/edu/caltech/ipac/firefly/server/servlets/AnyFileUpload.java

```diff
@@ -45,12 +46,13 @@
  * @version $Id: AnyFileUpload.java,v 1.3 2011/10/11 21:44:39 roby Exp $
  */
 public class AnyFileUpload extends BaseHttpServlet {
-    private static final Logger.LoggerImpl _LOG = Logger.getLogger();
-    public static final String DEST_PARAM  = "dest";
-    public static final String PRELOAD_PARAM = "preload";
-    public static final String FILE_TYPE = "type";
-    public static final String CACHE_KEY = "cacheKey";
-    private enum FileType {FITS, TABLE, REGION, XML, UNKNOWN}
+    private static final String DEST_PARAM  = "dest";
+    private static final String PRELOAD_PARAM = "preload";
```

Comment:
Just curious, what is the purpose of preload parameter? I saw it in firefly_client, but it unused.

Reply:
I think that is something @loitly did experimentally.  It was outside the scope of my work.